lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -48,7 +51,7 @@ def render_to_fragment(self, request, **kwargs):
 
         context = {
             'marketing_url': get_program_marketing_url(programs_config),
-            'programs': meter.engaged_programs,
+            'programs': Paginator(meter.engaged_programs, getattr(settings, 'ITEMS_VISIBLE_DASHBOARD', 10)).page(page),
```

Comment:
Hi @huynguyen-groove ,
Can you please update this varibable ITEMS_VISIBLE_DASHBOARD name to `COURSE_ITEMS_ON_LMS_DASHBOARD`